src/sentry/tasks/reports.py

```diff
@@ -355,23 +355,33 @@ def prepare_project_usage_outcomes(start__stop, project):
     data = raw_snql_query(query, referrer="reports.outcomes")["data"]
 
     return (
-        # accepted errors
+        # Accepted errors
         sum(
             row["total"]
             for row in data
             if row["category"] in DataCategory.error_categories()
             and row["outcome"] == Outcome.ACCEPTED
         ),
+        # Dropped errors
+        sum(
+            row["total"]
+            for row in data
+            if row["category"] in DataCategory.error_categories()
+            and row["outcome"] in (Outcome.FILTERED, Outcome.RATE_LIMITED)
```

Comment:
We don't combine filtered and dropped elsewhere when we show usage totals. Doing that here will make these numbers not add up to those shown on org stats and billing usage. Perhaps the title in the email could be `Dropped & Filtered`? Or this value should just be rate-limited events so that 'dropped' means the same things across the app.

Reply:
cc @doralchan @jeremybourque

Reply:
I think we should just ignore Filtered for the purposes of the email and show Outcome.RATE_LIMITED as Dropped so that it matches Dropped on the Org Stats page.